sklearn/preprocessing/data.py

```diff
@@ -1650,8 +1650,12 @@ class OneHotEncoder(BaseEstimator, TransformerMixin):
         Number of values per feature.
 
         - 'auto' : determine value range from training data.
-        - int : maximum value for all features.
-        - array : maximum value per feature.
+        - int : number of categorical values per feature. This should be set
```

Comment:
Maybe it would be better to be more explicit and say something like feature values are expected to be ``[0, ..., n_values - 1]`` ?